docs/en/notes/contribution_guide.md

```diff
@@ -1,31 +1,234 @@
-# Contributing to MMPose
+# How to Contribute to MMPOSE
 
-All kinds of contributions are welcome, including but not limited to the following.
+Welcome to join the MMPOSE community, we are committed to building cutting-edge computer vision foundational library. All kinds of contributions are welcomed, including but not limited to:
 
-- Fix typo or bugs
-- Add documentation or translate the documentation into other languages
-- Add new features and components
+- **Fix bugs**
+  1. If the modification involves significant changes, it's recommended to create an issue first that describes the error information and how to trigger the bug. Other developers will discuss it with you and propose a proper solution.
+  2. Fix the bug and add the corresponding unit test, submit the PR.
+- **Add new features or components**
+  1. If the new feature or module involves a large amount of code changes, we suggest you to submit an issue first, and we will confirm the necessity of the function with you.
+  2. Implement the new feature and add unit tests, submit the PR.
+- **Improve documentation or translation**
+  - If you find errors or incomplete documentation, please submit a PR directly.
 
-## Workflow
+```{note}
+- If you hope to contribute to MMPose 1.0, please create a new branch from dev-1.x and submit a PR to the dev-1.x branch.
+- If you are the author of papers in this field and would like to include your work to MMPose, please contact us. We will much appreciate your contribution.
+- If you hope to share your MMPose-based projects with the community at once, consider creating a PR to `Projects` directory, which will simplify the review process and bring in the projects as soon as possible. Checkout our [example project](/projects/example_project)
+- If you wish to join the MMPose developers, please feel free to contact us and we will invite you to join the MMPose developers group.
+```
+
+## Preparation
+
+The commands for processing pull requests are implemented using Git, and this chapter details Git Configuration and associated GitHub.
+
+### 1. Git Configuration
+
+First, you need to install Git and configure your Git username and email.
+
+```shell
+# view the Git version
+git --version
+```
+
+Second, check your Git Config.
+
+```shell
+# view the Git config
+git config --global --list
+```
+
+If `user.name` and `user.email` are not configured, you need to configure them.
 
-1. Fork and pull the latest MMPose repository
-2. Checkout a new branch (do not use master branch for PRs)
-3. Commit your changes
-4. Create a PR
+```shell
+git config --global user.name "Change your username here"
+git config --global user.email "Change your useremail here"
+```
+
+Finally, run the command in git bash or terminal to generate the key file. After the generation is successful, a `.ssh` file will appear in the user directory, and `id_rsa.pub` is the public key file.
```

Comment:
I suggest we remove ssh content and recommend contributors use http, which is easier.